src/content/learn/describing-the-ui.md

```diff
@@ -358,15 +358,15 @@ export default function PackingList() {
 
 <LearnMore path="/learn/conditional-rendering">
 
-Read **[Conditional Rendering](/learn/conditional-rendering)** to learn the different ways to render content conditionally.
+Baca **[Rendering Kondisional](/learn/conditional-rendering)** untuk mempelajari cara merender konten secara kondisional.
 
 </LearnMore>
 
-## Rendering lists {/*rendering-lists*/}
+## *Rendering* Daftar {/*rendering-lists*/}
```

Comment:
Hello @resir014, do you have any suggestions for this? I feel like this sentence is a little bit awkward, and another direct translation may not be understood by people (ex: "lis"), my suggestion is "Rendering Koleksi Data", wdyt

Reply:
```suggestion
## Merender List {/*rendering-lists*/}
```

We can use this. Also it's best to avoid using italic text in headings. @mustafamilyas